package.json

```diff
@@ -38,9 +38,10 @@
     "test-api": "mocha -R dot --recursive tests/api",
     "test-disablehttp": "DISABLE_HTTP=true mocha -R dot --recursive tests/disableHttp",
     "test-enforced": "USE_ACCESS_TOKEN=true mocha -R dot --recursive tests/enforceToken",
+    "test-cache-persp": "ENABLE_CACHE_PERSPECTIVE=true mocha -R dot --recursive tests/enableCache",
```

Comment:
/tests/testUtils.js has a function called "toggleOverride" -- since this feature toggle isn't used to instantiate the app server in index.js, you should be able to set and unset it directly in your test module, see example in tests/jobQueue/v1/bulkUpsert.js. That way you don't have to set up a separate package.json script for it and the tests can be counted by coveralls.